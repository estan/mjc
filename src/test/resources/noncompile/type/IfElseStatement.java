class IfElseStatementTest {
    public static void main(String[] args) {
        int i;
        int[] ia;
        boolean b;
        A cA;
        B cB;

        if (i) {} else {} // INVALID_CONDITION_TYPE
        if (ia) {} else {} // INVALID_CONDITION_TYPE
        //if (b) {} else {} // OK
        if (cA) {} else {} // INVALID_CONDITION_TYPE
    }
}
class A {}
class B {}
