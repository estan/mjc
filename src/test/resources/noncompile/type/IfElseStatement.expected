INVALID_CONDITION_TYPE
INVALID_CONDITION_TYPE
INVALID_CONDITION_TYPE
